tasks/agent.py

```diff
@@ -65,8 +65,8 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
 
     # remove all tags that are only available on debian distributions
     distname = platform.linux_distribution()[0].lower()
-    if distname not in ['debian', 'ubuntu']:
-        for ex in DEBIAN_ONLY_TAGS:
+    if distname not in ['debian', 'ubuntu', 'centos']:
```

Comment:
please add `redhat` to the list as well (see https://github.com/python/cpython/blob/v2.7.15/Lib/platform.py#L261-L264)

might be worth making this list a module-level constant also, up to you